recipes_source/recipes_index.rst

```diff
@@ -274,6 +274,13 @@ Recipes are bite-sized, actionable examples of how to use specific PyTorch featu
    :link: ../recipes/recipes/tuning_guide.html
    :tags: Model-Optimization
 
+.. customcarditem::
+   :header: Config Optimization on Intel® Xeon
```

Comment:
CPU launcher script for optimal performance on ...

Reply:
thanks for the suggestion. Done.